packages/core/docs/integrations/adyen.md

```diff
@@ -191,4 +191,11 @@ interface AdyenConfigBuilder {
 If the transaction is authorized, the server's controller for `payAndOrder`/`submitAdditionalPaymentDetails` will place an order in Commercetools and apply the `order` object to the response. Thanks to that, we have only one request from the client to both finalize/authorize a payment and make an order.
 
 ## Checkout.com
-Adyen's module isn't compatible with [Checkout.com's module](https://github.com/vuestorefront/checkout-com).
\ No newline at end of file
+Adyen's module isn't compatible with [Checkout.com's module](https://github.com/vuestorefront/checkout-com).
+
+## FAQ
+### Error: NotFound: URI not found: /<project_name>/carts/<cart_id>
+`ctApi.apiHost` property inside your `middleware.config.js` contains unnecessary path. It has to be `https://<SHOP_DOMAIN>.com/` instead of `https://<SHOP_DOMAIN>.com/<project_name>/graphql`
```

Comment:
```suggestion
`ctApi.apiHost` property inside your `middleware.config.js` contains wrong path. It should be `https://<SHOP_DOMAIN>.com/` instead of `https://<SHOP_DOMAIN>.com/<project_name>/graphql`
```